src/git.rs

```diff
@@ -91,6 +91,12 @@ pub fn pull<P: AsRef<Path>>(dir: &P, branch: &str) -> Result<bool> {
     )
     .context("failed pulling in git: merge")
 }
+/// Clone a full repository
+pub fn clone_full(dir: &str, url: &str) -> Result<bool> {
+    // Create a dummy path since we specify absolute path for cloning
+    let dummy_path = std::path::PathBuf::from("./");
```

Comment:
Can you `use std::path::PathBuf` at the top and then use `PathBuf::from("./");` here?